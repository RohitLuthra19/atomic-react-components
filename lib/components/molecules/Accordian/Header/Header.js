// @flow
/**
 *
 * Header
 *
 */
import React from 'react';
import styled from 'styled-components';
import styles from './Header.style';

const Header = () => {
  return (
    <>
      <span>Accordian</span>Header
    </>
  );
};

Header.defaultProps = {};

export default styled(Header)`
  ${styles};
`;

export { Header as HeaderVanilla };
