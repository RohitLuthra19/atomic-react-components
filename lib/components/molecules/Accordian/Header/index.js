// @flow
export { default } from './Header';
export { HeaderVanilla } from './Header';
