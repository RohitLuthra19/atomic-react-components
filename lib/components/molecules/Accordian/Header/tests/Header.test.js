import { shallow } from 'enzyme';
// import renderer from 'react-test-renderer';

import Header from '../index';

describe('<Header />', () => {
  let HeaderComponent = '';
  beforeEach(() => {
    HeaderComponent = shallow(<Header>Test</Header>);
  });

  test('should render correctly', () => {
    expect(HeaderComponent).toMatchSnapshot();
  });
});
