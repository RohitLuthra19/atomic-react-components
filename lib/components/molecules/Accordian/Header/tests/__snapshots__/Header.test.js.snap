// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<Header
  className="Header-sc-18g047y-0 FGjVP"
>
  Test
</Header>
`;
